examples/platform/silabs/efr32/BUILD.gn

```diff
@@ -141,7 +145,10 @@ config("efr32-common-config") {
 }
 
 source_set("efr32-common") {
-  deps = [ "${silabs_common_plat_dir}/provision:storage" ]
+  deps = [
+    "${chip_root}/src/tracing",
```

Comment:
Short answer yes, because of the define, but I'll look around to check if we could have a cleaner dependency that doesn't involve the registry source.

Reply:
Yup, not needed